src/Shared/Recruit.Vacancies.Client/Infrastructure/TableStore/TableStorageQueryStore.cs

```diff
@@ -19,8 +19,7 @@ internal sealed class TableStorageQueryStore : IQueryStore
         private const int DeleteBatchSize = 100;
         private const string PartitionKeyFieldName = nameof(TableEntity.PartitionKey);
         private readonly JsonSerializerSettings _jsonWriter = new JsonSerializerSettings
-        {
-            Formatting = Formatting.Indented,
```

Comment:
Eh?

Reply:
I am having issues when inserting the EmployerDashboard Projection. I get the Property value is too large storage exception. Hence, i am commenting out the Indendation , to reduce the size of the property value. It was discussed with Mark and Sachin yesterday.